sympy/parsing/c/c_parser.py

```diff
@@ -365,18 +356,18 @@ def transform_var_decl(self, node):
 
             except StopIteration:
 
-                if (node.type.kind == cin.TypeKind.INT):
-                    type = IntBaseType(String('integer'))
+                if node.type.kind in self._data_types["int"]:
+                    type = self._data_types["int"][node.type.kind]
                     value = Integer(0)
```

Comment:
Can you just remove the values here and from the return statement also?
I think it'll better to just return a Variable with its type without a default value when it was not given a default value initially.

Reply:
Ok